Tools/px_uploader.entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -eu
+
+/bin/px_uploader.py --udp-addr=192.168.200.101 --udp-port=14541 --port=/dev/px4serial --baud-flightstack=57600,115200,1000000,2000000 --baud-bootloader=2000000 --use-protocol-splitter-format px4_fmu-v5_ssrc-*.px4 px4_fmu-v5x_ssrc-*.px4 ssrc_saluki-v1_default-*.px4
+/bin/px4-post-update-init -a :14540
```

Comment:
I don't see a flight mode being passed here?

The logs you shared of testing indicate it picked outdoor flight mode:

```
2022/05/13 13:28:56 +-------------------------------+
2022/05/13 13:28:56 |  Target flightMode : outdoor  |
2022/05/13 13:28:56 |  Target airframe   : 4400     |
2022/05/13 13:28:56 +-------------------------------+
```

Is the outdoor mode the default? Seems to be:
https://github.com/tiiuae/px4-post-update-init/blob/ff4c6cf09607b7c1d9891546d535c536c0afbf68/main.go#L21

To make the effect of this command easier to understand, I think explicit is better than implicit, i.e.:

```console
$ /bin/px4-post-update-init -a :14540 -m outdoor
```

Is clearer in its effect (even though the argument is not strictly required by the flag default value of already being "outdoor")

Now thinking more of this, does this default mode break our most common case which is indoor flight?

I'm trying to understand the concept better, so another question :smile:: let's jump forward in time and imagine there's a flight mode selector in fog_hyper debug web UI. What should fog_hyper do when the flight mode is changed? Is it expected to run something like `$ docker run --rm --network=host --device=/dev/px4serial:/dev/px4serial --entrypoint="" px4-firmware px4-post-update-init -a :14540 -m SELECTED_FLIGHT_MODE` ?

Reply:
> To make the effect of this command easier to understand, I think explicit is better than implicit,

Reply:
> What should fog_hyper do when the flight mode is changed?
Running px4-firmware container like mentioned is doable, but there is also container image with only px4-post-update-init in ghcr.io

Reply:
> but there is also container image with only px4-post-update-init in ghcr.io

But that doesn't contain the flight mode config.txt files? Those are only baked-in in the `px4-firmware` container image.

And I think it's simpler if the hypervisor side only has to know the existence of this `px4-firmware` container image. The `px4-post-update-init` can be just an implementation detail that this `px4-firmware` is composed of :)

Reply:
> But that doesn't contain the flight mode config.txt files? Those are only baked-in in the px4-firmware container image.
Right, config files are only in px4-firmware image. So, I guess currently only way is to use that way you mentioned. I'm not sure if it is usable to copy config files from px4-firmware image to px4-post-update-init image, that would create kind of recursive loop between them..